src/applications/gi/reducers/calculator.js

```diff
@@ -47,14 +47,60 @@ export default function (state = INITIAL_STATE, action) {
       ].includes(field);
 
       if (isDollarAmount && !isFinite(value)) {
-      // Strip all non-numeric characters.
+        // Strip all non-numeric characters.
         convertedValue = +value.replace(/[^0-9.]+/g, '');
       }
 
-      const newState = {
+      let newState = {
         [field]: convertedValue,
       };
 
+      if (__BUILDTYPE__ !== 'production') {
+
+        if (field === 'yellowRibbonDegreeLevel') {
```

Comment:
When the degree level is updated, this updates

- the list of divisions options
- the value in the yellow ribbon contribution amount
- the selected division to the first option